src/widgets/adhole/AdHoleSummary.tsx

```diff
@@ -0,0 +1,102 @@
+import { Card, Center, Container, Stack, Text } from '@mantine/core';
+import { IconAd, IconBarrierBlock, IconPercentage, IconSearch, IconWorldWww } from '@tabler/icons';
+import { defineWidget } from '../helper';
+import { WidgetLoading } from '../loading';
+import { IWidget } from '../widgets';
+import { formatNumber } from '../../tools/client/math';
+import { useAdHoleSummeryQuery } from './query';
+
+const definition = defineWidget({
+  id: 'adhole-summary',
+  icon: IconAd,
+  options: {},
+  gridstack: {
+    minWidth: 2,
```

Comment:
Same as a prior comment, having the minWidth at 2 would cause text and icons to be cut off on medium-sized displays
```suggestion
    minWidth: 3,
```